addons/src/createNumberMask.js

```diff
@@ -16,12 +16,14 @@ export default function createNumberMask({
   thousandsSeparatorSymbol = comma,
   allowDecimal = false,
   decimalSymbol = period,
+  decimalSymbolAlt: decimalSymbolAltParam,
```

Comment:
here should be coma I guess, unless you reference undefined variable
```suggestion
  decimalSymbolAlt: comma,
```

Reply:
It is not assigning default value to `decimalSymbolAlt`. It's renaming variable in destructuring (in other words it's like `decimalSymbolAltParam = params.decimalSymbolAlt`)

Reply:
yes, sorry for that, my bad. Anyway good addition thanks, I found it useful for myself (put a star on it).
I'd maybe added a warning message, if ```decimalSymbolAlt === thousandsSeparatorSymbol```. 

For example this code won't work as expected.
```
    const mask = createNumberMask({
      allowDecimal: true,
      decimalSymbolAlt: ",",
      includeThousandsSeparator: false,
      prefix: "",
    });
```

unless I do this:
```
    const mask = createNumberMask({
      allowDecimal: true,
      decimalSymbolAlt: ",",
      includeThousandsSeparator: false,
      prefix: "",
      **thousandsSeparatorSymbol: "?",**
    });
```

Reply:
Thanks for this case! I didn't think about it, because I always used `thousandsSeparatorSymbol: " "`. I'll add a warning